parser/src/command/concern.rs

```diff
@@ -25,9 +25,16 @@ impl fmt::Display for ParseError {
 impl ConcernCommand {
     pub fn parse<'a>(input: &mut Tokenizer<'a>) -> Result<Option<Self>, Error<'a>> {
         let mut toks = input.clone();
-        if let Some(Token::Word(action @ ("concern" | "resolve"))) = toks.peek_token()? {
+        if let Some(Token::Word(mut action @ ("concern" | "resolve"))) = toks.peek_token()? {
             toks.next_token()?;
 
+            if action == "concern" {
```

Comment:
Hmm, what is someone tries to create a concern (sentence) that starts with the word "resolve"? 😆 Well, probably a too niche of an edge-case.

Reply:
I asked jieyouxu about it and I was told "that seems somewhat... strange xD", so left it. We can always add it later.

Reply:
Yeah, I thought that was even more niche, and maybe more of a "well don't do that" xD